### ISSUE TITLE
Captions and alt text display in material theme

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#2326]
**Describe the bug**
Here are a few more article display issues to make you aware of related to the way captions and alt text display in the material theme:
- Alt text (JATS <alt-text>) is visually displayed rather than being put into the alt attribute of the HTML img element. Is this an XSLT issue?
- There isn't any space after the fig div before the next p, which makes the caption (or currently the alt text) look like the start of the paragraph that follows it and makes it hard to read. Could the CSS add some space in there?

Also attaching a screenshot of the inspection pane. XML is available for download from the article.

**Janeway version**
1.3.9

**To Reproduce**
1. Go to https://journals.publishing.umich.edu/gs/article/id/855/
2. Scroll to figure 7.1.
4. See alt text and lack of space before next p

**Expected behavior**

**Screenshots**
![Screen Shot 2021-07-13 at 1 02 59 PM](https://user-images.githubusercontent.com/47217308/127695887-6f8cb6b0-0f06-45e4-9053-e00a2731bd2d.png)

**Comment by pgoussy** [comment]
Bumping this up because I just encountered some articles w/ image alt-text, and the text tagged with <alt-text> is still visually displaying in the HTML:
https://journals.publishing.umich.edu/gs/plugins/typesetting/preview_galley/article/2489/galley/1732/

**Comment by pgoussy** [comment]
Bumping this up yet again to point out that evidently it is not limited to the Material theme. The alt-text displays visibly with OLH as well--and unlike Material, the styling and alignment of the displayed alt-text appears identical to regular paragraph text, without any indication that it's part of the preceding figure/caption.

https://journals.publishing.umich.edu/ars/plugins/typesetting/preview_galley/article/3986/galley/2032/

![2023-03-29_16-17-03](https://user-images.githubusercontent.com/36245100/228657583-006b368a-4bca-4e56-997a-cb10c18679af.png)

**Comment by ajrbyers** [comment]
@pgoussy I’ll speak to the team tomorrow about adding a fix to liquid XSL for this. Permafix is in for the 1.5.1 release.

**Comment by pgoussy** [comment]
@ajrbyers Do you think a hotfix for this is feasible? Or will we need to wait until 1.5.1?

**Comment by mauromsl** [comment]
> @ajrbyers Do you think a hotfix for this is feasible? Or will we need to wait until 1.5.1?

Now that you have the Liquid XSLT plugin, it should be fine to add this to your install as soon as it is ready. I'm on the case
